gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/internal/IgorService.groovy

```diff
@@ -83,4 +86,23 @@ interface IgorService {
 
   @GET('/concourse/{buildMaster}/teams/{team}/pipelines/{pipeline}/resources')
   List<String> getConcourseResources(@Path("buildMaster") String buildMaster, @Path("team") String team, @Path("pipeline") String pipeline);
+
+  static class GoogleCloudBuildTrigger {
```

Comment:
If this is going to be used outside this class I'd recommend pulling it to it's own class.